astrobee/behaviors/inspection/src/inspection.cc

```diff
@@ -526,8 +526,16 @@ bool Inspection::GeneratePanoramaSurvey(geometry_msgs::PoseArray &points_panoram
   std::vector<PanoAttitude> orientations;
 
   // pano_orientations() doesn't support panos with non-zero center (not needed)
-  assert(pan_min_ == -pan_max_);
-  assert(tilt_min_ == -tilt_max_);
+  if (pan_min_ != -pan_max_) {
+    ROS_ERROR_STREAM(" Pan min: " << pan_min_ << " ; Pan max: " << pan_max_
+                                  << "; They are different! Making pan_min = pan_max");
+    pan_min_ = pan_max_;
```

Comment:
I'll fix this! Fatal makes the node crash which is really inconvenient because it implies that we have to restart isaac fsw... I can instead return failed survey generation and make the user resend the goal